### PR TITLE
Warn about using Commands in Dockerfile that are not supported by OCI.

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@nalind @TomSweeneyRedHat @pixdrift @umohnani8 PTAL

File: cmd/buildah/from.go [hunk 1]
```diff
@@ -211,6 +212,16 @@ func fromCmd(c *cli.Context) error {
 	}
 	namespaceOptions.AddOrReplace(usernsOption...)
 
+	format := c.String("Format")
+	switch format {
+	case "oci":
+		format = imagebuildah.OCIv1ImageFormat
+	case "docker":
+		format = imagebuildah.Dockerv2ImageFormat
+	default:
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/from.go]: Same comment as from another review, can we merge these definition out of image.go and imagebuildah and into common.go?

Comment (rhatdan) [reply cmd/buildah/from.go]: I want to get the other one merged first then I can clean this one up.

**Comment by rhatdan** [comment]
@rh-atomic-bot r=umohnani8

**Comment by rhatdan** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**